resources/puppetlabs/lein-ezbake/template/global/install.sh.erb

```diff
@@ -70,9 +71,15 @@ function task_service {
         else
             task install_source_rpm_systemd
         fi
-    elif [ "$OSFAMILY" = "Debian" ]; then
+      elif [ "$OSFAMILY" = "Debian" ]; then
+        sysv_codenames=("squeeze" "wheezy" "lucid" "precise" "trusty")
+        codename=`facter os.distro.codename`
```

Comment:
facter is only used in the task_service and task_all functions, which deb and rpm packaging never calls, either directly or indirectly. the source install does use this though i think.